src/Umbraco.Core/Constants-Conventions.cs

```diff
@@ -348,5 +348,10 @@ public static class RelationTypes
 
             // TODO: return a list of built in types so we can use that to prevent deletion in the uI
         }
+
+        public static class Udi
+        {
+            public const string StartUdi = "umb://";
```

Comment:
@patrickdemooij9 could this be named `Prefix` for simplicity as the class is named `Udi`?

Reply:
Updated to prefix as that is much clearer